internal/config/sidecar.go

```diff
@@ -18,8 +18,6 @@
 package config
 
 type AgentSidecar struct {
-	// Name string `yaml:"name" json:"name"`
-
 	// WatchPaths represents watch path list for backup
 	WatchPaths []string `yaml:"watch_paths" json:"watch_paths"`
```

Comment:
![badge](https://img.shields.io/badge/review-note-informational.svg?style=flat-square)
It should be single path `string` to specify ngt-index dir